test/e2e-conformance-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Script entry point.
+source $(dirname "$0")/e2e-secret-tests.sh
```

Comment:
do you need this at all?

Reply:
Not really, I should have been sourcing `vendor/knative.dev/test-infra/scripts/e2e-tests.sh` for the `success` function. I will fix that.